src/circuits/circuits.jl

```diff
@@ -193,14 +193,23 @@ end
 Build a random quantum circuit with `N` qubits and depth `depth`.
 """
 function randomcircuit(
-  N::Int,
-  depth::Int,
-  coupling_sequence::Vector{<:Any};
+  coupling_sequence::Vector,
```

Comment:
Could you explain this new interface?

Reply:
Nothing new, the number of qubits `N` was unnecessary here since this information is already contained in coupling sequence.

Reply:
I see, got it.